### PR TITLE
Feature: APP-2223 add apis needed for existing tokens

### PR DESCRIPTION
**Pull request by jamesej**
## Description

Added these apis:
- utils/tokens.ts getOwner()
- services/covalentAPI getTokenHoldersPaged(), getTotalHolders(), getTotalSupply()

The use of Covalent API means a .env file update is required. The PR updates workflows to enable use of this key in builds, and the key has been added as a secret in Github.

Task: [APP-2223](https://aragonassociation.atlassian.net/browse/APP-2223)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2223]: https://aragonassociation.atlassian.net/browse/APP-2223?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "big.js": "^6.2.1",
     "cpx": "^1.5.0",
     "date-fns": "^2.28.0",
+    "dotenv": "^16.1.3",
```
Comment (sepehr2github) [inline packages/web-app/package.json:41]: why did you add dotenv?

Comment (jamesej) [reply packages/web-app/package.json:41]: It's needed for Jest testing so that when you run the app code via a test, it can read stuff from the .env file. This is enabled by `setupFiles: ['dotenv/config']` in the `jest.config.js` file.

Comment (sepehr2github) [reply packages/web-app/package.json:41]: Ah. fair enough. thanks 👍

**Review by sepehr2github — APPROVED** [review]
Thanks
